src/v4/scripts/generate-dependency-map.js

```diff
@@ -0,0 +1,60 @@
+const dependencyTree = require('dependency-tree');
+const glob = require('glob');
+const Runner = require('jscodeshift/src/Runner');
+const path = require('path');
+const fs = require('fs-extra');
+const { execSync } = require('child_process');
+const rimraf = require('rimraf');
+const cpx = require('cpx');
+
+const coreDir = 'src/v4/core';
+const v4Path = path.resolve(__dirname, '../');
+
+/** @type {{ [coreImport: string]: string[]; }} */
+const deps = {};
+
+async function runTransform(options) {
```

Comment:
async functions aren't supported in Node 6. It'd be better to convert to TypeScript.